server.cc

```diff
@@ -125,7 +137,7 @@ static bool get_lucky(int probability)
     if (dist(rd) > probability) {
         return true;
     }
-    
+
```

Comment:
cosmetic changes should be done in cosmetic-only patches, that contain no functional change.